java/src/Ice/src/main/java/com/zeroc/Ice/_ObjectPrxFactoryMethods.java

```diff
@@ -4,114 +4,119 @@
 
 package com.zeroc.Ice;
 
+// TODO: This entire file is actually dead code. But instead of deleting it, I'm keeping it so that
```

Comment:
Yes, this file is literally completely dead-code.
It seems like someone started down the path I'm on, but never finished.

I'm already working on a follow-up PR to this one, which will make use of this class.
But for now, it's commented out to get things compiling.